backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -173,7 +173,7 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
 
             project.ProjectNumber = String.IsNullOrWhiteSpace(model.ProjectNumber) ? $"TEMP-{DateTime.UtcNow.Ticks:00000}" : model.ProjectNumber;
             project.Name = model.Description.Truncate(100);
-            project.Description = model.Description;
+            project.Description = model.Description + (String.IsNullOrWhiteSpace(model.Location) ? null : Environment.NewLine + model.Location);
```

Comment:
forget how c# appends strings. Just wanting to confirm that "test" + null doesn't return something awful like "testnull"

maybe for safety this should be "" instead of null?